review.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python
+
+import argparse, datetime, json, os, subprocess, requests
+
+
+def is_third_monday(d):
+    return d.weekday() == 0 and 15 <= d.day <= 21
+
+def find_monday(d, days=1, limit=10):
+    i = 0
+    while not is_third_monday(d):
+        d = d + datetime.timedelta(days=days)
+        i += 1
+        if i > limit:
+            return None
+    return i * days
+
+def days_from_third_monday(d):
+    if d.day < 15:
+        return find_monday(d)
+    elif d.day > 21:
+        return find_monday(d, -1)
+    else:
+        forward = find_monday(d)
+        if not forward:
+            return find_monday(d, -1)
+        return forward
+
+
+def find_shortnames(workstreams, month):
+    shortnames = []
+
+    for workstream in workstreams:
+        for standard in workstream["standards"]:
+            if month not in standard["review_draft_schedule"]:
+                continue
+            shortnames.append(href_to_shortname(standard["href"]))
+
+    return shortnames
+
+def href_to_shortname(href):
+    return href[len("https://"):href.index(".")]
+
+
+def maybe_create_prs(shortnames):
+    for shortname in shortnames:
+        os.chdir("../{}".format(shortname))
+        maybe_create_pr(shortname)
+        os.chdir(".")
+
+def maybe_create_pr(shortname):
+    subprocess.run(["git", "checkout", "master"], capture_output=True)
+    commits = subprocess.run(["git", "log", "--pretty=oneline", "--abbrev-commit", "--abbrev=8", "--max-count=40"], capture_output=True).stdout
```

Comment:
```suggestion
    commits = subprocess.run(["git", "log", "--format=%s", "--max-count=40"], capture_output=True).stdout
```

Printing only the subject line for each commit would simplify this loop a bit overall.